### ISSUE TITLE
[BUG] Unexpected native token data in `tokens_zksync.transfers`

### ISSUE THREAD

**Issue by lgingerich** [duneanalytics/spellbook#4970]
### Description
1. zkSync Era emits token transfer events for ETH, which I believe not all EVM chains do. This means that ETH transfers are included in the `tokens_zksync.transfers` for `token_standard` = `erc20`. There is also data in this table for `token_standard` = `native`, which is where I would expect all ETH data to be.
2. The symbol field has the string literal "{{native_symbol}}" for `token_standard` = `native`. This applies for at least zkSync and Arbitrum, but likely stems from an issue in the macro and therefore carries through to all chains.

### Current behavior
Behavior for both of the above problems can be seen in the below picture.
1. For the ETH contract address (0x000000000000000000000000000000000000800a), there is data for both the native and erc20 token standards.
2. Unexpected `symbol` = `{{native_symbol}}`
![image](https://github.com/duneanalytics/spellbook/assets/82065267/a3dfffe5-1b34-4cd6-8bd6-71f2dad0075a)

This can be reproduced for a single tx_hash like:
```
SELECT
    tx_hash,
    contract_address,
    token_standard,
    symbol
FROM tokens_zksync.transfers
WHERE tx_hash = 0xfddfaefa8449a9a1e12668554e8f0411fe1f1748f718b8974587ffae592261d1
```

### Expected behavior
1. I would expect all ETH transfers to only be in the `token_standard` = `native` results. 
2. Expect `symbol` to show the actual symbol, not `{{native_symbol}}`.

### Impacted model(s)

```
1. `tokens_zksync.transfers`
4. Potentially all `tokens_{chain}.transfers` tables
```

### Possible solution
1. I'm aware that this token transfers table is fairly new, so I'm open to suggestions on how to fix this behavior.
2. Seems likely that the string literal is being passed through the macro rather than the actual variable. Likely a simple fix.

**Comment by jeff-dude** [comment]
thank you for raising 🙏 

we haven't fully communicated out these new spells, but the goal was to have users start querying them and finding any potential bugs/enhancements/etc. -- this is a good head start.

fyi @aalan3 something we can look to fix. also, @lgingerich if you are interested in looking to submit a PR with a fix, we're happy to have ya contribute with us. let us know here

**Comment by lgingerich** [comment]
Looks like issue 2 has been fixed. I'll check out issue 1

**Comment by jeff-dude** [comment]
closing in favor of recent PRs to fix, such as #5212
